gpt4all-bindings/python/gpt4all/gpt4all.py

```diff
@@ -22,28 +22,20 @@ class GPT4All():
         model: Pointer to underlying C model.
     """
 
-    def __init__(self, model_name: str, model_path: str = None, model_type: str = None, allow_download=True):
+    def __init__(self, model_name: str, model_path: str = None,  model_type: str = None, allow_download=True):
```

Comment:
Minor detail, there is an extra space. Also, the spacing around the `=` is inconsistent.

Reply:
ah good catch, thanks!